platform-tests/src/test/java/org/junit/platform/console/tasks/TreeNodeTests.java

```diff
@@ -27,6 +28,16 @@
 	private static final int NUM_THREADS = 2;
 	private static final int ITEMS_PER_THREAD = 1000;
 
+	@Test
+	void caption() {
+		assertEquals("", TreeNode.createCaption(""));
+		assertEquals("@", TreeNode.createCaption("@"));
+		assertEquals("@.", TreeNode.createCaption("@\t"));
+		assertEquals("@..", TreeNode.createCaption("@\t\n"));
+		assertEquals("@...", TreeNode.createCaption("@\t\n\r"));
+		assertEquals("@".repeat(80) + "...", TreeNode.createCaption("@".repeat(1000)));
```

Comment:
❓ Is there anything special about an `@`?

Reply:
No. Just a character.